dwave/embedding/transforms.py

```diff
@@ -223,9 +229,16 @@ def embed_bqm(self, source_bqm, chain_strength = 1.0, smear_vartype = None):
         # add the offset
         target_bqm.add_offset(source_bqm.offset)
 
+        if chain_strength is None:
+            # Picked 1.415 as our prefactor as an avg of 1.33 (pegasus) 
+            # and 1.5 (chimera)
+            chain_strength = get_chain_strength(source_bqm, 1.415)
+
         if callable(chain_strength):
             chain_strength = chain_strength(source_bqm, self)
```

Comment:
Actually, on second read, I noticed this inconsistency in APIs of `get_chain_strength` and user-supplied `chain_strength`.

Users should be able to supply our "`get_chain_strength`" (or however we end up calling it) as the `chain_strength` callable argument. That means we need to specialize it by encoding the prefactor there (or at least make the prefactor a third kwarg, with a default value -- so users can use `partial` to derive their versions).

Then, the code above changes to:
```
if chain_strength is None:
    chain_strength = get_chain_strength
```